Quadriga/src/test/java/edu/asu/spring/quadriga/service/publicwebsite/impl/AboutTextManagerTest.java

```diff
@@ -0,0 +1,67 @@
+package edu.asu.spring.quadriga.service.publicwebsite.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.spring.quadriga.dao.impl.publicwebsite.AboutTextDAO;
+import edu.asu.spring.quadriga.dto.AboutTextDTO;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.impl.publicwebsite.AboutTextManager;
+
+public class AboutTextManagerTest {
+
+    @InjectMocks
+    private AboutTextManager aboutTextManager;
+
+    @Mock
+    private AboutTextDAO mockedAboutTextDAO;
+
+    private ArgumentCaptor<AboutTextDTO> argument;
+    private AboutTextDTO dto;
+
+    @Before
+    public void saveAbout() throws QuadrigaStorageException {
+        MockitoAnnotations.initMocks(this);
+        aboutTextManager.saveAbout("1st", "Title", "Description");
+        aboutTextManager.saveAbout("1st", "NewTitle", "NewDescription");
+
+    }
+
+    @Test
+    public void testSaveAboutCall() throws QuadrigaStorageException {
```

Comment:
what are you testing here? That you save a new about text object? Then this should be something like 'testSaveAboutNewDTO'.